jina/schemas/__init__.py

```diff
@@ -1,16 +1,20 @@
+from jina.schemas.gateway import schema_gateway
```

Comment:
is there any special reason to import the schema_gateway at the begining, rather than inside the `get_full_schema()`?

Reply:
no good reason, thanks for catching this